project_euler/problem_39/sol1.py

```diff
@@ -0,0 +1,41 @@
+"""
+If p is the perimeter of a right angle triangle with integral length sides,
+{a,b,c}, there are exactly three solutions for p = 120.
+{20,48,52}, {24,45,51}, {30,40,50}
+
+For which value of p ≤ 1000, is the number of solutions maximised?
+"""
+
+from typing import Dict
+
+
+def pythagorean_triple(max_perimeter: int) -> Dict:
+    """
+    Returns a dictionary with keys as the perimeter of a right angled triangle
+    and value as the number of corresponding triplets.
+    >>> pythagorean_triple(15)
+    {12: 1}
+    >>> pythagorean_triple(40)
+    {12: 1, 30: 1, 24: 1, 40: 1, 36: 1}
+    >>> pythagorean_triple(50)
+    {12: 1, 30: 1, 24: 1, 40: 1, 36: 1, 48: 1}
+    """
+    triplets = {}

```

Comment:
Triplets should be a https://docs.python.org/3/library/collections.html#collections.Counter

Reply:
Ok. I'll make the change